app/lib/sms-games/controllers/SGCreateFromMobileController.js

```diff
@@ -9,12 +9,11 @@ var requestHttp = require('request')
   , mobilecommons = rootRequire('mobilecommons')
   , smsHelper = rootRequire('app/lib/smsHelpers')
   , logger = rootRequire('app/lib/logger')
-  , competitiveStoryConfig = require('../config/competitive-stories')
   , connectionOperations = rootRequire('app/config/connectionOperations')
   , configModel = require('../models/sgGameCreateConfig')(connectionOperations)
   ;
 
-var gameConfig;
+var gameCreateConfig;
```

Comment:
Is this needed? `gameCreateConfig` only looks like it's used as a function parameter in `createGame()`.